packages/store-ui/src/Minicart/i18n/pt.ts

```diff
@@ -5,6 +5,11 @@ export default {
   'minicart.drawer.shipping-disclaimer':
     'Frete e impostos são calculados no checkout.',
   'minicart.drawer.go-checkout': 'FINALIZAR COMPRA',
+  'minicart.price.free': 'GRÁTIS',
```

Comment:
Make it uppercase with CSS `text-transform: uppercase;`

```suggestion
  'minicart.price.free': 'Grátis',
```